explorer/testdata/assert/assert.carbon

```diff
@@ -0,0 +1,18 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// RUN: %{explorer} %s 2>&1 | \
+// RUN:   %{FileCheck} --match-full-lines --allow-unused-prefixes=false %s
+// RUN: %{explorer} --parser_debug --trace_file=- %s 2>&1 | \
+// RUN:   %{FileCheck} --match-full-lines --allow-unused-prefixes %s
+// AUTOUPDATE: %{explorer} %s
+// CHECK: ASSERTION ERROR: {{.*}}/explorer/testdata/assert/assert.carbon:16: "HALLO WELT"
+
+package ExplorerTest api;
+
+fn Main() -> i32 {
+  __intrinsic_assert(true, "HALLO WELT");
```

Comment:
I think we should try to avoid using intrinsics directly in user code, so let's wrap this in an `Assert` function in the prelude.

Reply:
You are right!

Reply:
Done